.travis.yml

```diff
@@ -15,7 +15,7 @@ script:
 - wget -q https://github.com/HL7/fhir-ig-publisher/releases/download/1.1.25/publisher.jar -O input-cache/publisher.jar
 - JAVA_TOOL_OPTIONS=-Dfile.encoding=UTF-8
 - java -jar input-cache/publisher.jar -ig ig.ini
-- java -jar validator.jar -ig output/validator-de.bbmri.fhir.pack -version 4.0.1 input/examples
+- java -jar validator.jar -tx n/a -ig output/validator-de.bbmri.fhir.pack -ig de.basisprofil.r4#0.9.6 -version 4.0.1 input/examples
```

Comment:
Why do we need to disable the terminology server here?
Why is the German Basisprofil not a dependency of our IG?

Reply:
Why is the German Basisprofil not a dependency of our IG? -> It is, but we use the local build files, not the package for validation, so no dependencies are resolved.

Reply:
Why do we need to disable the terminology server here? -> The new DKTK-Histologie example contains an ICD-O-3 topography () code. The Terminology server claims this is not a valid code in ICD-O-3 ...
(*FAILURE* validating input/examples/Observation-dktk-histo.json:  error:1 warn:1 info:0

  Error @ Observation.value.ofType(CodeableConcept).coding[0] (line 25, col8) : The code "8720/2" is not valid in the system urn:oid:2.16.840.1.113883.6.43.1; The code provided (urn:oid:2.16.840.1.113883.6.43.1#8720/2) is not valid in the value set 'All codes known to the system' for "urn:oid:2.16.840.1.113883.6.43.1#8720/2")